### DISCUSSION TITLE
How to install tremor with pnpm

### DISCUSSION THREAD

**Discussion by Byron2016** [tremorlabs/tremor-npm#920] [General]
Maybe are there a tutorial to install **tremor** using **pnpm**?
With **npm** you can use:
`npx @tremor/cli@latest init`

but what about **pnpm**?

**Comment by iamtouha** [comment]
Hey @Byron2016, To install tremor with `pnpm` use-
```bash
pnpm dlx @tremor/cli@latest init
```
